### DISCUSSION TITLE
GitHub Student Discount Not Working After Applying for 7 times in a row

### DISCUSSION THREAD

**Discussion by ghost** [community/community#52647] [GitHub Education]
Hello, I tried applying for GitHub Student several times. The first time i was told to use my webcam instead of a image. I did that but then it said that i am far away from my school. Then I saw that the device i was using had some location issue so i switched to a laptop and used a webcam to capture my student ID. I really do not like this. If you can't give students a little pass as they can't afford the subscription, why do you even have that at your homepage.

Image:-
![Screenshot 2023-04-12 215225](https://user-images.githubusercontent.com/102423760/231522630-acf3a8e1-9549-4a7f-9275-64cf886e63e5.png)

**Comment by ghost** [comment]
Can anyone help

**Comment by samus-aran** [comment]
Hi @RamanujGhosal 

When students receive this rejection reason (or lack of) we would usually suggest that you provide additional pieces of evidences. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Hope this helps! Let us know how you get along 👍
